src/FMData.Rest/FileMakerRestClient.cs

```diff
@@ -98,6 +100,19 @@ public FileMakerRestClient(
         {
             _authTokenProvider = authTokenProvider;
             _useNewClientForContainers = useNewClientForContainers;
+            switch (_authTokenProvider.ConnectionInfo?.RestTargetVersion)
+            {
+                case RestTargetVersion.v1:
+                case null:
+                    _targetVersion = "v1";
+                    break;
+                case RestTargetVersion.v2:
+                    _targetVersion = "v2";
+                    break;
+                default:
+                    _targetVersion = "vLatest";
+                    break;
```

Comment:
```suggestion
                case RestTargetVersion.v1:
                    _targetVersion = "v1";
                    break;
                case RestTargetVersion.v2:
                    _targetVersion = "v2";
                    break;
                case RestTargetVersion.vLatest:
                    _targetVersion = "vLatest";
                    break;                    
                default:
                    _targetVersion = "v1";
                    break;
```

What do you think of this instead? I think the default value for the readonly string is good, I think this keeps the intent clear that the default behavior is `v1` unless specifically opting into another value?